File: examples/indexer/go/SearchAssetsName.go
package main

import (
	"context"
	"encoding/json"
	"fmt"

	"github.com/algorand/go-algorand-sdk/client/v2/indexer"
)

// indexer host
const indexerAddress = "http://localhost:8980"
const indexerToken = ""

// query parameters
var assetName = "DevDocsCoin"

func main() {

	// Create an indexer client
	indexerClient, err := indexer.MakeClient(indexerAddress, indexerToken)
	if err != nil {
		return
	}

	// Lookup asset
	_, result, err := indexerClient.SearchForAssets().Name(assetName).Do(context.Background())

	// Search asset by name
	JSON, err := json.MarshalIndent(result, "", "\t")
	fmt.Printf(string(JSON) + "\n")
}

File: examples/indexer/python/search_assets_name.py
# search_assets_name
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

# instantiate indexer client
myindexer = indexer.IndexerClient(indexer_token="", indexer_address="http://localhost:8980")

response = myindexer.search_assets(
    name="DevDocsCoin")
print("Asset Name Info: " + json.dumps(response, indent=2, sort_keys=True))
